tests/integration/IntegrationTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Integration;
+
+use function array_filter;
+use function array_merge;
+use function array_walk;
+use function assert;
+use function escapeshellarg;
+use function explode;
+use function file_exists;
+
+use function file_get_contents;
+use function implode;
+
+use const LC_ALL;
+
+use phpDocumentor\Guides\ApplicationTestCase;
+use phpDocumentor\Guides\Cli\Command\Run;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\ExpectationFailedException;
+
+use function setlocale;
+use function str_ends_with;
+use function str_replace;
+
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Finder\Finder as SymfonyFinder;
+
+use function system;
+
+use function trim;
+
+class IntegrationTest extends ApplicationTestCase
```

Comment:
Should we make this class "abstract" (like TestCase is also abstract)? So it is clear that is is meant for extending?

Reply:
Sorry, mixed it up.